eldritch/eldritch.py

```diff
@@ -379,6 +379,8 @@ def get_interrupts(self, event):
       if nearby_monsters:
         interrupts.append(events.EvadeOrFightAll(event.character, nearby_monsters))
     interrupts += sum([char.get_interrupts(event, self) for char in self.characters], [])
+    global_interrupts = [glob.get_interrupt(event, self) for glob in self.globals() if glob]
+    interrupts += [interrupt for interrupt in global_interrupts if interrupt]
```

Comment:
Probably doesn't affect the overall runtime, but `interrupts.extend` is more pythonic.